app/src/main/java/com/erwanlbp/calculit/activity/MainActivity.java

```diff
@@ -81,16 +97,30 @@ private void startPrintResults(int userAnswer) {
         intent.putExtra(AnswerActivity.USER_ANSWER, userAnswer);
         intent.putExtra(GameConfig.CONFIG_CORRECT_RESULT, correctResult);
         intent.putExtra(GameConfig.CONFIG_LEVEL, gameConfig.getLevel());
-        startActivityForResult(intent, ActivityCode.SHOW_RESULT);
+        startActivityForResult(intent, ActivityCode.RQ_SHOW_RESULT);
     }
 
     public void selectDifficultyActivity(View view) {
         final Intent intent = new Intent(this, SelectDifficultyActivity.class);
-        startActivityForResult(intent, ActivityCode.SELECT_DIFFICULTY);
+        startActivityForResult(intent, ActivityCode.RQ_SELECT_DIFFICULTY);
     }
 
     private void createGameConfig(final Intent data) {
         final Difficulty difficulty = Difficulty.parse(data.getIntExtra(SelectDifficultyActivity.DIFFICULTY, Difficulty.EASY.getTimeToPrint()));
         gameConfig = new GameConfig(difficulty);
     }
+
+    public void startPrintHighScores(View view) {
+        final Intent intent = new Intent(this, PrintHighScoresActivity.class);
+        intent.putExtra(User.USER_PSEUDO, user.getPseudo());
+        intent.putExtra(User.USER_HIGH_SCORE_EASY, user.getHighScoreEasy());
+        intent.putExtra(User.USER_HIGH_SCORE_MEDIUM, user.getHighScoreMedium());
+        intent.putExtra(User.USER_HIGH_SCORE_HARD, user.getHighScoreHard());
```

Comment:
On peut mettre un ticket dessus : https://stackoverflow.com/questions/2139134/how-to-send-an-object-from-one-android-activity-to-another-using-intents 
C'est pour passer un objet et pas 36 trucs un par un, ca demande un peu de taff par contre